src/routes/project/TaskDescription.svelte

```diff
@@ -3,27 +3,94 @@
 	import Backdrop from '$lib/components/Backdrop.svelte'
 	import Panel from '$lib/components/Panel.svelte'
 	import { escape } from '$lib/directives/escape'
+	import { Character } from '$lib/model/character'
+	import WaitingImage from '$lib/components/WaitingImage.svelte'
 	export let isOpen: boolean = false
 	export let task: Task | null
 	export let close: VoidFunction
+	export let assignee: Character | undefined
 </script>
 
 <Backdrop {isOpen}>
 	<Panel>
 		<div class="container" use:escape on:escape={close}>
-			<h3 class="title">{task?.name}</h3>
+			<h3>{task?.name}</h3>
 			<div class="description">{task?.description}</div>
-			<div class="additional-data">
-				<p class="status">Status: {task?.status}</p>
-				<p class="estimate">Estimated time: {task?.estimate}</p>
+			<div class="resume">
+				<div class="resume__statuses">
+					<p class="resume__status">
+						<span class="resume__title">Status:</span>
+						<span class="resume__value">{task?.status}</span>
```

Comment:
Let's use a map for that. Task status is a enum, so you can easily use a map (aka dictionary) for that. And also 'inprogress' and 'todo' are not proper English words (although TODO is kinda, but not really).